src/serac/numerics/equation_solver.cpp

```diff
@@ -106,6 +106,12 @@ class NewtonSolver : public mfem::NewtonSolver {
         mfem::out << '\n';
       }
 
+      if (norm != norm) {
```

Comment:
I'm guessing one of these should have been `norm_goal`

Reply:
its a nan check, maybe too old school

Reply:
std::isnan has not always been reliable for me.  maybe there are some optimization that turn it off, so I go old school by default.

Reply:
interesting.. learn something new every day. Leave as-is and put a comment?

Reply:
nm the error message explains.. carry on